### DISCUSSION TITLE
Why do you have so many kinds of metadata / tagging / symbols / dates?

### DISCUSSION THREAD

**Discussion by ih8snow** [BryanJenksCommunity/FAQ#67] [Q&A] [answered]
Hi Bryan,

I'm just doing an autopsy of your templates and tag taxonomy. 😎
There are some really great inputs I already copied into my vault.

Can you give me some hints on why you have: 
- the type of note (great idea to include that in the filename and as linked tag), i.e. % and
- in the metadata the icon tags for i.e. podcast?

Looks a bit like a duplication?
I see that you can use % as a linked page, similar to "linked tags", but a search via tags would give you the same results?

I'm creating an area in my vault with training material (for trainings I provide) and I created a few MOC to link the content. Now I'm at the point to think about the template design and I wonder if I should use both ideas. In concert with the MOCs (and a folder) I have four ways to see all these notes...maybe a bit too much?

Ayn thoughts on this?

Kind regards.

Jörn

**Comment by tallguyjenks** [comment, marked as answer]
I have a variety of ways of classifying information so that the system can scale. Not knowing new features, plugins and how they may interact with what I do not I use a variety of methods for sorting, sifting, and organizing.

the titles `% 2021-04-17 Podcast Name` allows me

-  to sort all inputs by special symbol when sorted by name
- it groups them together in the file pane
- and when I look through those groups they're sorted chronologically
- when using the quick switching i only have to type 1 symbol: `%` to immediately filter everything down to JUST podcasts
- then I can further filter by using either the ISO 8601 date stamp, OR the actual verbose title. 
- even over the years as I get more notes for more items, this will easily allow me to cut through the noise.
- i link them to `[[%]]` because this allows me to easily see a collection now with either the native query syntax or data view to see a list of all items linked to `[[%]]` plus it will also group the nodes on the global graph or I can use local graph.
- i use tags to keep track of status processing as well as grouping
  - when I have `#📥️/📚️/🟥️` i know that I have a book literature note:  `#📥️/📚️` that is currently not processed `/🟥️` and I can click for any layer of granularity
    - `#📥️` for all inputs/literature notes
    - `/📚️` for all books
    - `/🟥️` for all of those items that are not processed.

many ways to get the job done, but I want a swiss army knife of approaches so I'm prepared for any new plugins, systems, tools, and scaling my second brain receives of undergoes.

**Reply by ih8snow** [reply]
Great,
thanks for the details. That's very helpful for, as I just started the vault.

**Reply by ih8snow** [reply]
I just learned that some characters in the beginning of a file name are not really a good idea when you use
- Windows
- Markdown Links in Obsidian

So I created some links with % and was wondering why Windows was asking for an App to open this internal Obsidian links.

The same happened when I used ( in the beginning of the filename. 

When I used + it was finally the expected result and it worked for me.

When I used Wiki-links in Obsidian I did not experience any problems with the characters.

As I need to stay with MD-Links (less export problems) I need to dig a bit deeper to identify the characters I can use in this context.

Just my two cents :-)

Jörn

**Reply by tallguyjenks** [reply]
This is why I use *nix based systems. Linux and Mac don't have that problem. 🤣
